client/src/pages/locations/GeoLocation.js

```diff
@@ -29,14 +29,16 @@ const GeoLocation = ({
   lat,
   lng,
   editable,
-  setValues,
+  setFieldValue,
   setFieldTouched,
   isSubmitting,
-  displayType
+  displayType,
+  locationFormat,
+  values
```

Comment:
Right now I don't find it trivial to follow where the displayedCoordinate is coming from, or when to use conversion.  I would also think that we don't need all formik form values in here (see the values prop).

Actually, as the form also has the displayedCoordinate, we can also just pass it down to the GeoLocation component. And always use all 3 coordinates (lat, lng, displayedCoordinate) in all components from locations/GeoLocation.
That way we prevent converting from one format to the other in too many components.

I would think the update of the coordinate values and the conversion should only happen when one changes the position in the map or changes the lat/lng/displayedCoordinate fields.

I have made a short proposal for a refactpring which might simplify things - for now for the MGRSFormField (LatLonFormField is still to be done), see here: https://github.com/NCI-Agency/anet/commit/be08d9b1e9635dc1496dfe45d58b97a3e00a101a
@cemalettin-simsoft what do you think about it?

We could also replace the 3 properties (lat, lng, displayedCoordinate) with one property, called coordinates, containing an object with the mentioned 3 keys.